### PR TITLE
AC-1475 add SCIM instructions

### PR DESCRIPTION
**Pull request by James-Burba**
### Support Docs Update Checklist

Note: when you open a pull request, your changes are published to https://staging.sparkpost.com/docs.

Please take a moment to check each of these steps for your pull request:

- [ ] Give your pull request a meaningful name.
- [ ] Include a description of your changes in the pull request.
- [ ] _SparkPost staff only_: Check that your article appears [here](https://staging.sparkpost.com/docs) and looks correct.
- [ ] Check the links in your article.
- [ ] Use lowercase filenames.

### CODE REVIEW

**Comment by bkemper** [comment]
@James-Burba Ping me when this is ready to be merged.  I'll have to manually deploy it.

File: support/my-account-and-profile/scim.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: "SCIM User Provisioning Through Okta"
+description: "Learn how to enable SCIM user provisioning through Okta for your SparkPost account."
+---
+
+**This integration with Okta is currently under development and is not available to all customers yet. Contact to learn more.**
```
Comment (dan2305) [inline support/my-account-and-profile/scim.md:6]: contact who?

File: support/my-account-and-profile/scim.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- SAML SSO must be configured and enabled for your account. See instructions [here](https://www.sparkpost.com/docs/my-account-and-profile/sso/).
+- Only admin users can generate SCIM access tokens.
+
+## Configuration in SparkPost
+1. From your SparkPost dashboard, navigate to your Account Settings page by clicking on your username at the upper right corner of the screen.
+1. Under "Single Sign-On" click "Generate SCIM token".
+1. Your token will be displayed. Copy it - you won't be able to see it again (don't worry, if you lose it you can generate a new one).
+
+## Configuration in Okta
+1. From the Okta admin console, naviate to "Applications" and select the SparkPost app.
```
Comment (dan2305) [inline support/my-account-and-profile/scim.md:25]: navigate

File: support/my-account-and-profile/scim.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+1. Select the "General" tab. Under "App Settings" check the "Enable SCIM Provisioning" box.
+1. Select the "Provisioning" tab, and from the "Integration" menu select "Edit".
+1. Enter `https://api.sparkpost.com/api/v1/users/scim/v2` in the "SCIM connector base URL" field.
```
Comment (dan2305) [inline support/my-account-and-profile/scim.md:28]: should we have a note for eu as well?

File: support/my-account-and-profile/scim.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+1. Enter `email` in the "Unique identifier field for users"
+1. Under "Supported provisioning actions, check the following boxes:
```
Comment (dan2305) [inline support/my-account-and-profile/scim.md:30]: missing `"`

**Pull request merged**